### PR TITLE
Testimonial section updated

### PR DESCRIPTION
**Pull request by Pragati-3003**
I have updated the testimonial section .
Screenshot 2023-05-30 020505
Kindly inform me if you want any other changes.

### CODE REVIEW

**Comment by varun-singhh** [comment]
@Pragati-3003 can you add a video/gif so that I can see how it looks. also fix conflicting issues

**Comment by Pragati-3003** [comment]
> @Pragati-3003 can you add a video/gif so that I can see how it looks. also fix conflicting issues


https://github.com/Deveimer/vovoca-frontend/assets/132778037/e6a3b119-c6b1-441a-b7f6-7d44432272d9

If you wanna any other changes then please inform me.

**Comment by varun-singhh** [comment]
@Pragati-3003 fix conflicting issue. Design looks good but We don't want transformation effect and hovering effect. It's just without any these just static design looks good. No need of adding hovering and transformation on this it has no use case.

**Comment by Pragati-3003** [comment]
> @Pragati-3003 fix conflicting issue. Design looks good but We don't want transformation effect and hovering effect. It's just without any these just static design looks good. No need of adding hovering and transformation on this it has no use case.

okie , I'll modify and solve merge conflicts.

**Comment by Pragati-3003** [comment]
> @Pragati-3003 fix conflicting issue. Design looks good but We don't want transformation effect and hovering effect. It's just without any these just static design looks good. No need of adding hovering and transformation on this it has no use case.

I have modified the section ad per your requirements, please review and merge it.

**Comment by kaustubhai** [comment]
image

Still conflicting @Pragati-3003

**Comment by Pragati-3003** [comment]
> image
> 
> Still conflicting @Pragati-3003

Resolved all the conflicts ,please check once again.

File: package.json
Comment (kaustubhai) [inline package.json:1]: Why am I seeing changes in package.json and package-lock?

File: styles/Testimonial.module.css [hunk 1]
```diff
@@ -33,21 +35,30 @@
...
     ),
     linear-gradient(160deg, #434b54 1%, #2a2c2f 45%, #191b1e 75%);
   background-clip: padding-box, border-box;
   background-origin: padding-box, border-box;
-  box-shadow: 1.875rem 1.5625rem 1.375rem rgb(21 22 24 / 24%),
-    -4px -4px 15px rgb(195 200 205 / 4%);
+ 
+  /* box-shadow: rgba(169, 169, 171, 0.15) 0px 2px 5px 0px, rgba(210, 205, 205, 0.05) 0px 1px 1px 0px; */
+  /* box-shadow: rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px; */
+  /* box-shadow: rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px; */
```
Comment (kaustubhai) [inline styles/Testimonial.module.css]: remove the commented code

File: styles/Testimonial.module.css [hunk 2]
```diff
@@ -78,18 +89,19 @@
...
+
 .testimonial__content > h5 {
   color: #d7e1ec;
   line-height: 1.7;
   font-weight: 500;
   letter-spacing: 0;
 }
 
-.testimonial__content > p {
+.testimonial__content  p {
```
Comment (kaustubhai) [inline styles/Testimonial.module.css]: is this part of your change? you are altering the current code here? please update me with the impact

**Review by kaustubhai — CHANGES_REQUESTED** [review]

File: styles/Testimonial.module.css (hunk 1 above)
Comment (Pragati-3003) [reply styles/Testimonial.module.css]: okie

**Comment by kaustubhai** [comment]
@Pragati-3003 you deleted the package-lock completely. Do the following if you need help with your changes:
1. `git checkout Deveimer:Development`
2. `git pull origin Deveimer:Development`
3. `git checkout Pragati-3003:vovoca-14_2`
3. Go to [this](https://github.com/Deveimer/vovoca-frontend/pull/85/files) page, see only the changes that you want to make and copy paste them
4. Commit and Push
5. Create a new PR from that new branch.

Hope it helps, happy to help more

**Comment by Pragati-3003** [comment]
> @Pragati-3003 you deleted the package-lock completely. Do the following if you need help with your changes:
> 
> 1. `git checkout Deveimer:Development`
> 2. `git pull origin Deveimer:Development`
> 3. `git checkout Pragati-3003:vovoca-14_2`
> 4. Go to [this](https://github.com/Deveimer/vovoca-frontend/pull/85/files) page, see only the changes that you want to make and copy paste them
> 5. Commit and Push
> 6. Create a new PR from that new branch.
> 
> Hope it helps, happy to help more

Thanks for the explanation.
I have created a new pr #125

**Pull request closed without merging**